code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -224,13 +230,26 @@
 		if("hiss", "hisses")
 			var/M = handle_emote_param(param)
 
-			if(!muzzled)
-				message = "<B>[src]</B> hisses[M ? " at [M]" : ""]."
-				playsound(loc, 'sound/effects/unathihiss.ogg', 50, 1, frequency = get_age_pitch()) //Credit to Jamius (freesound.org) for the sound.
-				m_type = 2
-			else
-				message = "<B>[src]</B> makes a weak hissing noise."
-				m_type = 2
+			if(isunathi(src))
+				if(!muzzled)
```

Comment:
Best to flip these.
Check the muzzled first so you can catch the makes a weak noise.
Then check if they are either unathi or vox. Can even just make the calling of the actual sound and message of hissing once.
So something like this:
```
if(!muzzled)
  var/sound_file
  if(vox)
    sound_file = X
  else if(uanthi)
    sound_file = Y
  play_sound
....
```